src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/mha_single_token.cpp

```diff
@@ -59,19 +62,36 @@ void cvt_copy(TA* dst, TB* src, size_t n) {
         mm256_uni_storeu_ps(dst + i, vb);
     }
 #elif defined(OPENVINO_ARCH_ARM64)
-    if (std::is_same<TA, float>::value && std::is_same<TB, float>::value) {
-        for (; i + vec_len_f32_neon <= n; i += vec_len_f32_neon) {
-            float32x4_t vb1 = __vld1q_f32(src + i);
-            __vst1q_f32(dst + i, vb1);
+#if defined(HAVE_SVE)
+    auto _dst = reinterpret_cast<float32_t*>(dst);
+    size_t inc = vec_len_f32_sve;
+    svbool_t pg = svptrue_b32();
+
+    while (i < n) {
+        if (n - i < vec_len_f32_sve) {
+            inc = n - i;
+            pg = svwhilelt_b32(0, static_cast<int>(inc));
         }
+        svfloat32_t b1 = svld1_f32(pg, src + i);
+        svst1_f32(pg, _dst + i, b1);
+        i += inc;
     }
+#else
 #if defined(__ARM_FEATURE_FP16_VECTOR_ARITHMETIC)
     if (std::is_same<TA, ov::float16>::value && std::is_same<TB, ov::float16>::value) {
         for (; i + vec_len_f16_neon <= n; i += vec_len_f16_neon) {
             auto vb1 = vld1q_f16(reinterpret_cast<const float16_t*>(src + i));
             vst1q_f16(reinterpret_cast<float16_t*>(dst + i), vb1);
         }
     }
+#else
+    if (std::is_same<TA, float>::value && std::is_same<TB, float>::value) {
+        for (; i + vec_len_f32_neon <= n; i += vec_len_f32_neon) {
+            float32x4_t vb1 = __vld1q_f32(src + i);
+            __vst1q_f32(dst + i, vb1);
+        }
+    }
+#endif
```

Comment:
Couple of tests fails with NEON isa (https://github.com/openvinotoolkit/openvino/actions/runs/12079060334/job/34240356815?pr=27273).
 I suspect the reason is that change. Please change it to 
```suggestion
#else
    if (std::is_same<TA, float>::value && std::is_same<TB, float>::value) {
        for (; i + vec_len_f32_neon <= n; i += vec_len_f32_neon) {
            float32x4_t vb1 = __vld1q_f32(src + i);
            __vst1q_f32(dst + i, vb1);
        }
    }
#if defined(__ARM_FEATURE_FP16_VECTOR_ARITHMETIC)    
    if (std::is_same<TA, ov::float16>::value && std::is_same<TB, ov::float16>::value) {
        for (; i + vec_len_f16_neon <= n; i += vec_len_f16_neon) {
            auto vb1 = vld1q_f16(reinterpret_cast<const float16_t*>(src + i));
            vst1q_f16(reinterpret_cast<float16_t*>(dst + i), vb1);
        }
    }
#endif
```

Reply:
Thanks for the suggestion. Added in latest commit.